src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -11,40 +11,38 @@
 # UMPIRE
 ################################
 if (UMPIRE_DIR)
-  include(cmake/thirdparty/FindUmpire.cmake)
-  blt_register_library( NAME umpire
-                        INCLUDES ${UMPIRE_INCLUDE_DIRS}
-                        TREAT_INCLUDES_AS_SYSTEM ON
-                        LIBRARIES umpire )
+    include(cmake/thirdparty/FindUmpire.cmake)
+    blt_register_library( NAME      umpire
+                          INCLUDES  ${UMPIRE_INCLUDE_DIRS}
+                          LIBRARIES umpire
+                          TREAT_INCLUDES_AS_SYSTEM ON)
 else()
-  message(STATUS "Umpire support is OFF")
+    message(STATUS "Umpire support is OFF")
 endif()
 
+
 ################################
 # RAJA
 ################################
 if (RAJA_DIR)
-  include(cmake/thirdparty/FindRAJA.cmake)
-  blt_register_library( NAME raja
-                        INCLUDES ${RAJA_INCLUDE_DIR}
-                        TREAT_INCLUDES_AS_SYSTEM ON
-                        LIBRARIES ${RAJA_LIB_DIR}/libRAJA.a )
+    include(cmake/thirdparty/FindRAJA.cmake)
+    blt_register_library( NAME      raja
+                          INCLUDES  ${RAJA_INCLUDE_DIR}
+                          LIBRARIES ${RAJA_LIB_DIR}/libRAJA.a
+                          TREAT_INCLUDES_AS_SYSTEM ON)
 else()
-  message(STATUS "RAJA support is OFF" )
+    message(STATUS "RAJA support is OFF" )
 endif()
 
+
 ################################
 # Conduit
 ################################
 if (CONDUIT_DIR)
     include(cmake/thirdparty/FindConduit.cmake)
-    blt_register_library( NAME conduit
-                          INCLUDES ${CONDUIT_INCLUDE_DIRS}
-                          LIBRARIES conduit
-                          TREAT_INCLUDES_AS_SYSTEM ON)
-    blt_register_library( NAME conduit_relay
-                          INCLUDES ${CONDUIT_INCLUDE_DIRS}
-                          LIBRARIES conduit_relay
+    blt_register_library( NAME      conduit
+                          INCLUDES  ${CONDUIT_INCLUDE_DIRS}
```

Comment:
you shouldn't have to set the includes, but it also wont hurt.

Reply:
Apparently the registered target "conduit" was conflicting with your under the hood exported target "conduit" and caused info to be lost.  I removed the registered target and switched completely to your convience target "conduit::conduit".  This solved the fact that it wasn't inheriting things properly.  This does lose the system header flag though.

Reply:
ahh ok, yes I don't see how to easily propgate treat as system.

If folks see annoying warnings from Conduit --  I'l just have to fix them. I know they exist on Windows but I have tried to keep them at bay on other platforms.